metron-interface/metron-alerts/e2e/login/login.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LoginPage } from './login.po';
+
+describe('login to application', function() {
+    let page: LoginPage;
+
+    beforeEach(() => {
+        page = new LoginPage();
+    });
+
+    it('should display error message for invalid credentials', () => {
```

Comment:
Ok, that's what I thought.  There is a dependency on Full Dev to run the E2E tests.

Do we have to do this?  Does it make more sense to mock the REST API, so the E2E tests can be run without this dependency?  It seems to be that it is going to be far easier to fully test border conditions and such if the E2E tests run against a mocked REST API.  Or is that crazy talk?  

But assuming we leave it as-is, then we should update the steps in the README for running the E2E test suite.  We should have an explicit step to start Full Dev.